core/services/flux_monitor_test.go

```diff
@@ -248,11 +237,21 @@ func TestPollingDeviationChecker_StartStop(t *testing.T) {
 	ethClient.On("SubscribeToLogs", mock.Anything, mock.Anything).
 		Return(fakeSubscription(), nil)
 
-	// Start() with no delay to speed up test and polling.
 	rm := new(mocks.RunManager)
+	fetcher := new(mocks.Fetcher)
 	checker, err := services.NewPollingDeviationChecker(initr, rm, fetcher, time.Millisecond)
 	require.NoError(t, err)
 
+	// Set up fetcher to mark when polled
+	started := make(chan struct{})
+	fetcher.On("Fetch").Return(decimal.NewFromFloat(100.0), nil).Maybe().Run(func(mock.Arguments) {
+		select {
+		case started <- struct{}{}:
+		default:
```

Comment:
This suggests that this test has potentially two outcomes right?

Reply:
That was placed there because in the older implementation, we didn't want to block on `Initialize` but we wanted to assert that `Start` polled. It's no longer needed with the collapse of `Initialize` into `Start`, and has been removed.